.github/actions/run-e2e-tests/action.yml

```diff
@@ -14,6 +14,11 @@ inputs:
   dsi_password:
     required: true
     type: string
+  environment:
+    required: true
+    type: string
+    default: "E2E"
```

Comment:
my confusion was that the docs mention this should be `dev`, `test`, `staging` or `production` so I was questioning if `E2E` is a valid value here and within pipeline

Reply:
I've modified the example to hopefully explain further, hope that's sufficient?

I think I understand what you mean about ASPNETCORE_ENVIRONMENT variable now, but (if I'm understanding you correctly), there's the possibility where they're running in separate shells + the `ASPNETCORE_ENVIRONMENT` hasn't been exported right? So having a separate variable should hopefully just ensure it gets captured. I mean I could just merge the two if you think that's better anyway, but I thought having separate ones would just make it a bit more explicit.